src/components/entity/show.vue

```diff
@@ -157,6 +154,16 @@ const requestDelete = () => {
 };
 
 const branchData = modalData();
+
```

Comment:
get resourceStates from useRequestData() to be able to use dataExists and replace entity.dataExists in the 2 places it is used. 

`const { initiallyLoading, dataExists } = resourceStates([project, entity]);`

entity.loading --> initiallyLoading